deps/build.jl

```diff
@@ -13,6 +13,9 @@ module DefaultDeps
     if !isdefined(@__MODULE__, :ROOTENV)
         const ROOTENV = joinpath(Main.condadir, MINICONDA_VERSION)
     end
+    if !isdefined(@__MODULE__, :USE_MINIFORGE)
+        const USE_MINIFORGE = "0"
```

Comment:
I guess we might eventually make this the default, but I agree that it should be experimental to start with.

Reply:
Seems like it should be a boolean, though.